tests/utils/create-e2e-fixture.ts

```diff
@@ -349,24 +357,27 @@ export async function deploySiteWithBuildbot(
   newZip.addLocalFolder(isolatedFixtureRoot, '', (entry) => {
     if (
       // don't include node_modules / .git / publish dir in zip
-      entry.startsWith('node_modules') ||
-      entry.startsWith('.git') ||
+      entry.includes('node_modules') ||
+      entry.includes('.git') ||
```

Comment:
new fixture has nested directory with `node_modules`, so this ensures we skip zipping those